### ISSUE TITLE
Create Accounts to sync favorites

### ISSUE THREAD

**Issue by cullub** [cullub/Tabs-Lite#54]
This will require a good bit of both app and backend work.  We'll have to create the concept of "accounts" on the server side, make the server interactive (more or less), and store lists of favorites (and playlists?) server side to enable syncing.

App side, we'll have to create a UI for account login and status.  Maybe integrate with google accounts?  Then sync favorites and playlists each time they're updated.

Also sync server->app maybe each time the app is started?  Maybe keep a version number to track updates.  Think about what to do when the user is signed in to multiple devices and edits from both devices.

Make sure this doesn't impact startup speed and doesn't impact offline functionality.

**Comment by cullub** [comment]
This should happen after #64

**Comment by cullub** [comment]
Update: paid for site hosting so can now start on this.

**Comment by AndydeCleyre** [comment]
Maybe related to or superseded by #153?
